nomos-da/kzgrs/src/kzg.rs

```diff
@@ -24,7 +24,7 @@ pub fn commit_polynomial(
         .map(|(commitment, _)| commitment)
 }
 
-/// Compute a witness polynomial in that satisfies `witness(x) = (f(x)-v)/(x-u)`
+/// Compute a quotient polynomial 'q' that satisfies `q(x) = (f(x)-v)/(x-u)`
```

Comment:
In the context of KZG (Kate-Zaverucha-Goldberg) polynomial commitments, the concept of a witness polynomial is used to verify the evaluation of a polynomial at a specific point. Lets keep the original comment.